### PR TITLE
Changes to Account App - social media models and changes to profile model

### PR DESCRIPTION
**Pull request by colleenobrien3**
- Related Issue (include '#'): #506 

- Description of changes made:
Social media type and profile social media models created, profile model edited as much as possible for now

- Is the feature complete/bug resolved/etc..: will have to further change profile model once other app models are complete and seed data is ready

- Any known bugs/strange behavior: location relationship is not in profile model yet, see above. this is so that the frontend functionality for creating a user is still there

- Is there specific feedback you would like on these changes:

- Screenshot(s):

### CODE REVIEW

File: api/accounts/views.py [hunk 1]
```diff
@@ -57,21 +58,45 @@ class ProfileList(generics.ListAPIView):
...
+class SocialMediaTypeList(generics.ListCreateAPIView):
+    queryset = SocialMediaType.objects.all()
+    serializer_class = SocialMediaTypeSerializer
+    permissions_classes = (permissions.IsAuthenticated)
+
+
+class SocialMediaTypeDetail(generics.RetrieveUpdateDestroyAPIView):
+    queryset = SocialMediaType.objects.all()
+    serializer_class = SocialMediaTypeSerializer
+    permissions_classes = (permissions.IsAuthenticated)
```
[the remark this answers is not in the document]
Comment (colleenobrien3) [reply api/accounts/views.py:102]: Cool thanks, I'll look into this once I finish editing the models for the forum app and adding the serializers, views, and urls

**Comment by vmipfdr** [comment]
Looking at the Profile Model, are we sure we want "Topics, Image, Activities" as the only required fields instead of legal things like "Age, First Name, and Last Name"?

Screen Shot 2020-04-08 at 2 57 11 PM

**Comment by colleenobrien3** [comment]
@vmipfdr is your comment implying you're changing it or is it asking me to change it? also not sure we are on the same branch/commit because I'm not getting that.

File: api/accounts/views.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (ZakkMan) [reply api/accounts/views.py:106]: @colleenobrien3 Are you going to do this in a separate PR?

[the remark this answers is not in the document]
Comment (colleenobrien3) [reply api/accounts/views.py]: @ZakkMan I guess I did not fully understand before. I have removed them now though.

**Pull request merged**